### ISSUE TITLE
Game is crashing with v1.14.8.46 - DX11

### ISSUE THREAD

**Issue by sourlemon75** [Soreepeong/XivAlexander#293]
Hi, my game is crashing with XIVAlexander installed. I have uninstalled and reinstalled the latest version v1.14.8.46. I haven't enabled the framerate lock nor the CPU features added. I am running the game with Dalamud plugins as well. I haven't encountered these fatal crashes before. I used the previous version v1.14.8.45. See log for details.

![XIVAlexander](https://user-images.githubusercontent.com/97267656/148480981-976bfec4-f725-4963-8169-9753fc642934.PNG)

[XivAlexander_20220107_101636_627.log](https://github.com/Soreepeong/XivAlexander/files/7825891/XivAlexander_20220107_101636_627.log)

**Comment by sourlemon75** [comment]
I am using Gshade as well. I have used both these as installation options, but both were still crashing. 
![image](https://user-images.githubusercontent.com/97267656/148481834-43e90c85-ee73-4928-a063-db40068533f9.png)

**Comment by DAOWAce** [comment]
Started getting this crash today too as well after updating the opcodes provided in the other issue.  May just be a coincidence though.

Also use Dalamud, GShade like others, but I also use SpecialK global injector (which triggered the crash on injection).  Don't use any of the new features in Alexander.

**Comment by Soreepeong** [comment]
Can you try using XivAlexander without any other addons (Dalamud, GShade, etc.) and see whether it works fine? Also, you might want to try integrity check using FFXIVQuickLauncher. That error "A fatal DirectX error has occurred.(11000002)" happens when a texture resource is corrupted, and if you aren't using XivAlexander modding, then it probably got nothing to do with XivAlexander.

**Comment by sourlemon75** [comment]
Ah it must be a texture mod I have installed. Running into no crashes for now. Thank you.

**Comment by sourlemon75** [comment]
The game just crashed again. I was running the game without Dalamud and GShade. I was flying over Elpis, as usual, to check for crashes. IDK what is in this particular area that causes the crashes. I have uninstalled every texture mod I used by using the start over option in Textools. I also ran the Integrity Check in XIVLauncher. 

[XivAlexander_20220107_170634_805.log](https://github.com/Soreepeong/XivAlexander/files/7827749/XivAlexander_20220107_170634_805.log)
